ramls/calendar.raml

```diff
@@ -174,32 +174,17 @@ resourceTypes:
                 description: "Internal server error, e.g. due to misconfiguration"
 
       /calculateopening:
-        is: [language]
         get:
+          is: [language]
           queryParameters:
-            startDate:
-              displayName: startDate
+            requestedDate:
+              displayName: requestedDate
+              description: "requested due date"
```

Comment:
This is not necessarily a due date, the calendar is not aware of what the purpose of this request is.

Reply:
fixed